packages/bus/package.json

```diff
@@ -40,14 +38,19 @@
   "devDependencies": {
     "@accelint/typescript-config": "workspace:*",
     "@accelint/vitest-config": "workspace:*",
+    "@types/react": "^19.1.9",
     "esbuild-fix-imports-plugin": "1.0.21",
+    "react": "^19.0.0",
     "tsup": "8.5.0",
     "vitest": "2.1.3",
     "vitest-broadcast-channel-mock": "0.1.0"
   },
   "dependencies": {
     "typescript": "^5.8.3"
   },
+  "optionalDependencies": {
```

Comment:
Why optional? Because most things in the package don't rely on it? And if you're using the React bits you probably already have React?

Reply:
The react hook is opt in regarding usage, meaning you can install and use `@accelint/bus` without React. Putting `react` as an optional dependency reflects that.